### DISCUSSION TITLE
`IgxFilterOptions` filter via **multiple** keys

### DISCUSSION THREAD

**Discussion by CaedenPH** [IgniteUI/igniteui-angular#12045] [Ideas]
```
const fo = new IgxFilterOptions();
    fo.key = 'make';
    if (!this.searchCar) {
      return fo; 
    }
    fo.inputValue = this.searchCar;
    return fo;
```
This is my code to filter via just the `make` key. How can I filter via multiple keys? Ie the make *and* model of a car

**Comment by zdrawku** [comment]
Filtering by multiple keys is not supported by the IgxFilter directive. I am marking this discussion with category "Idea" in order to consider it as a feature request in the future.
